drv/lpc55-gpio/Cargo.toml

```diff
@@ -9,7 +9,14 @@ lpc55-pac = "0.3.0"
 num-traits = { version = "0.2.12", default-features = false }
 drv-lpc55-gpio-api = {path = "../lpc55-gpio-api"}
 drv-lpc55-syscon-api = {path = "../lpc55-syscon-api"}
-lpc55-iocon-gen = {path = "../lpc55-iocon-gen"}
+idol-runtime = {git = "https://github.com/oxidecomputer/idolatry.git"}
+zerocopy = "0.6.1"
+ssmarshal = { version = "1", default-features = false }
```

Comment:
You can remove `ssmarshal`, `serde`, and `unwrap_lite`; the first two are unused, and the latter is now re-exported by `userlib`.